src/frequenz/dispatch/_dispatcher.py

```diff
@@ -95,6 +126,46 @@ async def run():
                     case _ as unhandled:
                         assert_never(unhandled)
         ```
+    Example: Creating a new dispatch and then modifying it. Note that this uses
+    the lower-level `Client` class to create and update the dispatch.
```

Comment:
Did you check how this documentation renders? I recommend using a shorter title anyway, and adding a longer description as part of the example:

```suggestion
    Example: Creating a new dispatch and then modifying it
    
        Note that this uses the lower-level `Client` class to create and update the dispatch.
```

Reply:
Also pending, it should be a quick fix.